common/util.go

```diff
@@ -654,3 +658,20 @@ func FormatList(list []string, conjunction string) string {
 func BoolToPointer(b bool) *bool {
 	return &b
 }
+
+// Sanitize text to match confusables and remove diacritics
+var transformer = transform.Chain(norm.NFD, transform.RemoveFunc(isMn), norm.NFC)
+
+func isMn(r rune) bool {
```

Comment:
what is this function name, and what is the use of this function?

Reply:
I just checked and it appears that RemoveFunc is depreciated anyways, but it's so the transformer knows whether or not to replace the char or not.